package.json

```diff
@@ -157,6 +159,7 @@
 		"leasot": "^13.3.0",
 		"mocha": "^10.4.0",
 		"mocha-jenkins-reporter": "^0.4.7",
+		"node-yaml-parser": "^0.0.9",
```

Comment:
The document links don't work when I install from .vsix . I think you will need to list this as a `dependency` instead of a `devDependency` in order to get it to work.

Reply:
Sure. You're right!